### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by ragangithub**
- Added RSpec as a gem to your project.
- Created a Request spec file for all  controllers.
- For each action checked:
  - If response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes the correct placeholder text in the response body' do
+      get users_path
+      expect(response.body).to include('Here is a list of users')
+    end
   end
```
Comment (prolajumokeoni) [inline spec/requests/users_spec.rb:19]: - [ ] Kindly add tests for show actions for User controller, this is currently missing in your project, see example below ⬇️  your tests should be inside the block  👍🏿 

````ruby
describe 'GET /show' do
  ...
end
````

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @ragangithub 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- index and show actions for  Post controllers are tested ✅
- index   actions for User controller is tested ✅



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- Added tests for show actions for User controller,

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**